src/include/cell.h

```diff
@@ -256,6 +256,9 @@ struct __wt_cell_unpack_delta_int {
     uint32_t __len;
     WT_CELL_UNPACK_KV key;
     WT_CELL_UNPACK_ADDR value;
+
+#define WT_DELTA_INT_IS_DELETE 0x01u
```

Comment:
This is unused?

Reply:
It is used - [here](https://github.com/wiredtiger/wiredtiger/pull/11627/files#diff-cad28305e3ada31c13e31189c328eb6ae2a9623df878744cd22c119ef6a93b9dR130)